### ISSUE TITLE
Connect: refactor connect-popup

### ISSUE THREAD

**Issue by tsusanka** [trezor/trezor-suite#4363]
### 1) We need redesign. That means we need to instruct the Product team to redesign.

- [x] #5050
- [ ] Create new design.

### 2) Decide what browsers we support.
- [x] Discuss with Product team and QA.

### 3) Start writing views
- [ ] Generate the same views in Suite monorepo.
- [ ] Make a "hello world".
- [ ] Apply the new design.
- [ ] make it translatable

### 4) Resolve connect issues
- [ ] [Issues labeled Popup](https://github.com/trezor/trezor-suite/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Aconnect-popup)

### 5) Add new data analytics
- [x] Add new data analytics, moved to https://github.com/trezor/trezor-suite/issues/4771

**Comment by tsusanka** [comment]
Just a small note that the sooner we add the data analytics the better as this is very important for some future decisions. It is probably clear but I wanted to stress that out. We can also create a new issue if that would help (as you wish @hynek-jina).

**Comment by hynek-jina** [comment]
I agree. So here is the related analytics issue: https://github.com/trezor/trezor-suite/issues/4771

**Comment by mroz22** [comment]
popup is now dead 🪂
